src/components/CuratorDashboard/PendingRequestManager/PendingRequestTabs/CuratorSubmission/SentRequestsSubmission.tsx

```
@@ -0,0 +1,394 @@
+																	<span className='mt-2 cursor-pointer text-[14px] font-medium text-[#1B61FF] hover:text-[#1B61FF]'>Read More</span>
+																</div>
+															</div>
+															{request?.status === EChildbountySubmissionStatus.OUTDATED && (
+																<>
+																	<Alert
+																		showIcon={true}
+																		message={'This proposal is outdated'}
+																		className='mx-4 mb-2'
+																	/>
+																</>
+															)}
```

Biome

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)